<!DOCTYPE html>
<html lang="en" dir="ltr"><head>
  
                           
     


<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="generator" content="Hugo 0.110.0">
<title>GnomAD API Query Tool | B치rbara Bitarello</title>


<meta property="twitter:site" content="@dudutchy">
<meta property="twitter:creator" content="@dudutchy">







  
    
  
<meta name="description" content="If you&#39;ve been trying to find an API query tool for gnomAD, you found it!">


<meta property="og:site_name" content="B치rbara Bitarello">
<meta property="og:title" content="GnomAD API Query Tool | B치rbara Bitarello">
<meta property="og:description" content="If you&#39;ve been trying to find an API query tool for gnomAD, you found it!" />
<meta property="og:type" content="page" />
<meta property="og:url" content="/blog/gnomad-api-query/" />
<meta property="og:locale" content="en">




    
        <meta property="og:image" content="/blog/sidebar-featured.jpg" >
        <meta property="twitter:card" content="summary_large_image">
        <meta name="twitter:image" content="/blog/sidebar-featured.jpg" >
    
    
  <meta itemprop="name" content="GnomAD API Query Tool">
<meta itemprop="description" content="Tl; dr, I love to tell a good story. If you don&rsquo;t wanna read that, no worries. Just skip ahead or get straight to business!
Now, to the story.
I have been working on a project where I was using population allelic frequencies for about 6,000 SNPs (single nucleotide polymorphisms) for CEU (North Americans from UTAH with European ancestry) individuals from the 1000 Genomes Project. We have our own copy of this dataset in our computer, and it&rsquo;s relatively straightforward to get those frequencies from vcf files."><meta itemprop="datePublished" content="2020-07-21T00:00:00+00:00" />
<meta itemprop="dateModified" content="2020-07-21T00:00:00+00:00" />
<meta itemprop="wordCount" content="469">
<meta itemprop="keywords" content="software,genome browser,bioinformatics,computational biology,JSON,API,allele frequencies,SFS,tutorial,collaboration," />
  
<script type="application/javascript">
var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
var doNotTrack = (dnt == "1" || dnt == "yes");
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'UA-173463950-1', 'auto');
	ga('set', 'anonymizeIp', true);
	ga('send', 'pageview');
}
</script>
  
<script type="application/javascript">
var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
var doNotTrack = (dnt == "1" || dnt == "yes");
if (!doNotTrack) {
	window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
	ga('create', 'UA-173463950-1', 'auto');
	ga('set', 'anonymizeIp', true);
	ga('send', 'pageview');
}
</script>
<script async src='https://www.google-analytics.com/analytics.js'></script>

  <!--[if IE]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  <link rel="shortcut icon" href="/img/dna-solid.svg" type="image/x-icon">
  <link rel="icon" href="/img/dna-solid.svg" type="image/x-icon">
  
  
  <link rel="stylesheet" href="/style.main.min.92dfcd41e6de8ee9bdf4891b6db7aba055136f3e07bb0945cad731e21d441835.css" integrity="sha256-kt/NQebejum99IkbbberoFUTbz4HuwlFytcx4h1EGDU=" media="screen">
  
  
  <script src="/panelset.min.ed1ac24b6e16f4e2481e3d1d098ae66f5bc77438aef619e6e266d8ac5b00dc72.js" type="text/javascript"></script>
  
  
  <script src="/main.min.6407e1256d3f3d21ac8def46942ac52e585729b4c0a5e1f8acc2633c9154b7e1.js" type="text/javascript"></script>
</head>
<body>
      <div class="grid-container">
<header class="site-header pt4 pb2 mb4 bb b--transparent ph5 headroom z-max" role="banner">
  <nav class="site-nav db dt-l w-100" role="navigation">
    <a class="site-brand db dtc-l v-mid link no-underline w-100 w-33-l tc tl-l" href="/" title="Home">
      <img src="/img/dna-solid.svg" class="dib db-l h2 w-auto" alt="B치rbara Bitarello">
    </a>
    <div class="site-links db dtc-l v-mid w-100 w-47-l tc tr-l mt3 mt0-l ttu tracked">
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/about/" title="About us">about</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/people/" title="People">people</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/project/" title="Projects">projects</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/teaching/" title="Teaching">teaching</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/talk/" title="Talks, Interviews, etc">talks/press</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 active" href="/blog/" title="Blog">blog</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/publication/" title="Publications">publications</a>
      
      
    </div>
  </nav>
</header>

<main class="page-main pa4" role="main">
  <section class="page-content mw7 center">
    <article class="post-content pa0 ph4-l">
      <header class="post-header">
        <h1 class="f1 lh-solid measure-narrow mb3 fw4">GnomAD API Query Tool</h1>
        <h4 class="f4 mt0 mb4 lh-title measure">If you&#39;ve been trying to find an API query tool for gnomAD, you found it!</h4>
        
        <p class="f7 db mv0 ttu">July 21, 2020</p>

      

      </header>
      <section class="post-body pt5 pb4">
        <p>Tl; dr, I love to tell a good story. If you don&rsquo;t wanna read that, no worries. Just 
<a href="#a-brief-tutorial">skip ahead</a> or get 
<a href="https://github.com/crankysparrow/computationalbio" target="_blank" rel="noopener">straight to business</a>!</p>
<p>Now, to the story.</p>
<p>I have been working on a project where I was using population allelic frequencies for about 6,000 SNPs (single nucleotide polymorphisms) for CEU (North Americans from UTAH with European ancestry) individuals from the 
<a href="https://www.internationalgenome.org/faq/what-do-your-population-codes-ceu-or-tsi-mean/" target="_blank" rel="noopener">1000 Genomes Project</a>. We have our own copy of this dataset in our computer, and it&rsquo;s relatively straightforward to get those frequencies from vcf files. All good.</p>
<p>Then, I decided I should check whether CEU individuals were a good proxy for allelic frequencies in Europeans (excluding Finnish, which due to their interesting demographic history are usually excluded for this kind of analysis). I wanted to do this by comparing the allelic frequencies for CEU from the 1000 Genomes (404 individuals) to the tens of thousands &rsquo;non-Finnish European&rsquo; samples from the the Genome Aggregation Database (
<a href="https://gnomad.broadinstitute.org/" target="_blank" rel="noopener">gnomAD</a>). Hence, I went to their website and thought to myself that downloading their entire dataset and calculating allelic frequencies did not strike me as elegant. Neigther did the idea of manually quering ~6,000 SNP IDs on the gnomAD browser (which is great if you only have a few SNPs to look at). No batch submission option was available.</p>
<p>There must be a well-established gnomAD API query tool available, right? Wrong! No such tool. Of course, I did find some solutions people made here and there, but none of them did exactly what I wanted.</p>
<p>Enters 
<a href="https://michelleenos.com" target="_blank" rel="noopener">Michelle Enos</a>, who is an awesome web developper and is entirely foreign to the field of computational biology. I explained what I needed and they made the tool I needed in javascript (which I am not fluent in). Modest, @crankysparrow thought this wasn&rsquo;t much, &lsquo;a fun little project&rsquo;. But looking at biostars <sup id="fnref:1"><a href="#fn:1" class="footnote-ref" role="doc-noteref">1</a></sup>, I begged to differ: we 
<a href="https://www.biostars.org/p/375279/" target="_blank" rel="noopener"><em>NEED</em> this</a>!</p>
<p>Even cooler, after they did the hard work, I was able to tweek the script to include some options that computational biologists would appreciate. Feedback is appreciated.</p>
<p>So here it is, from us, to you. A gnomAD API query tool that gathers gnomad-genomes or gnomad-exomes allelic frequencies for a list of SNP IDs for a population defined by the user. 
<a href="https://github.com/crankysparrow/computationalbio" target="_blank" rel="noopener">Link here</a>. Enjoy!</p>




<h2 id="heading">
  <a href="#heading"><svg class="anchor-symbol" aria-hidden="true" height="26" width="26" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
      <path d="M0 0h24v24H0z" fill="currentColor"></path>
      <path d="M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76.0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71.0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71.0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76.0 5-2.24 5-5s-2.24-5-5-5z"></path>
    </svg></a>
</h2>




<h2 id="a-brief-tutorial">A brief tutorial:
  <a href="#a-brief-tutorial"><svg class="anchor-symbol" aria-hidden="true" height="26" width="26" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
      <path d="M0 0h24v24H0z" fill="currentColor"></path>
      <path d="M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76.0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71.0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71.0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76.0 5-2.24 5-5s-2.24-5-5-5z"></path>
    </svg></a>
</h2>
<p>Requirements:</p>
<p>*node must be installed</p>
<p>*command line</p>
<p>*npm must be installed</p>
<p><em>Note</em>: instructions available at the 
<a href="https://github.com/crankysparrow/computationalbio" target="_blank" rel="noopener">repo</a>.</p>




<h2 id="input-file">Input file
  <a href="#input-file"><svg class="anchor-symbol" aria-hidden="true" height="26" width="26" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
      <path d="M0 0h24v24H0z" fill="currentColor"></path>
      <path d="M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76.0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71.0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71.0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76.0 5-2.24 5-5s-2.24-5-5-5z"></path>
    </svg></a>
</h2>
<p>This should be a .json file with the SNP IDs you would like to query. IF you have a regular text file with one ID per line, you can use our 
<a href="https://github.com/crankysparrow/computationalbio/blob/master/scripts/readIDs.js" target="_blank" rel="noopener">conversion tool</a>:</p>
<pre><code>node scripts/readIDs.js --input path/&lt;your_text_file_with_SNP_ids&gt;.txt
</code></pre>
<p>This will produce an output file with the same name, but ending in &lsquo;.json&rsquo;. Then you can run:</p>
<pre><code>node scripts/gnomad_query.js --input input/&lt;our_SNP_IDs&gt;.json --pop 'NFE' --db 'GNOMAD'
</code></pre>
<p>Enjoy!</p>
<div class="footnotes" role="doc-endnotes">
<hr>
<ol>
<li id="fn:1">
<p>Biostars is the stackoverflow of computational biologists.&#160;<a href="#fnref:1" class="footnote-backref" role="doc-backlink">&#x21a9;&#xfe0e;</a></p>
</li>
</ol>
</div>

        
        <details closed class="f6 fw7 input-reset">
  <dl class="f6 lh-copy">
    <dt class="fw7">Posted on:</dt>
    <dd class="fw5 ml0">July 21, 2020</dd>
  </dl>
  <dl class="f6 lh-copy">
    <dt class="fw7">Length:</dt>
    <dd class="fw5 ml0">3 minute read, 469 words</dd>
  </dl>
  
  
  
  <dl class="f6 lh-copy">
    <dt class="fw7">Tags:</dt>
    <dd class="fw5 ml0"> <a href="/tags/software">software</a>  <a href="/tags/genome-browser">genome browser</a>  <a href="/tags/bioinformatics">bioinformatics</a>  <a href="/tags/computational-biology">computational biology</a>  <a href="/tags/json">JSON</a>  <a href="/tags/api">API</a>  <a href="/tags/allele-frequencies">allele frequencies</a>  <a href="/tags/sfs">SFS</a>  <a href="/tags/tutorial">tutorial</a>  <a href="/tags/collaboration">collaboration</a> </dd>
  </dl>
  
  <dl class="f6 lh-copy">
    <dt class="fw7">See Also:</dt>
    
    <dd class="fw5 ml0"><a href="/blog/ldpred2/">LDpred2: Better, Faster, Stronger</a></dd>
    
  </dl>
</details>

      </section>
      <footer class="post-footer">
        <div class="post-pagination dt w-100 mt4 mb2">
  
  
    <a class="prev dtc pr2 tl v-top fw6"
    href="/blog/how-lion-king/">&larr; How The Lion King turned me into a scientist</a>
  
  
  
    <a class="next dtc pl2 tr v-top fw6"
    href="/blog/ldpred2/">LDpred2: Better, Faster, Stronger &rarr;</a>
  
</div>

      </footer>
    </article>
    
  </section>
</main>
<footer class="site-footer pv4 bt b--transparent ph5" role="contentinfo">
  <nav class="db dt-l w-100">
    <p class="site-copyright f7 db dtc-l v-mid w-100 w-33-l tc tl-l pv2 pv0-l mv0 lh-copy">
      &copy; 2024 B치rbara Bitarello, Anywhere
      <span class="middot-divider"></span>
      Made with <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/hugo-apero/" rel="dct:source">Hugo Ap칠ro</a></span>.
      <br />
      
Based on <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/formspree/blogophonic-hugo" rel="dct:source">Blogophonic</a></span> by <a xmlns:cc="http://creativecommons.org/ns#" href="https://formspree.io" property="cc:attributionName" rel="cc:attributionURL">Formspree</a>.
    </p>
    
    <div class="site-social-links db dtc-l v-mid w-100 w-33-l tc pv2 pv0-l mv0">
      <div class="social-icon-links" aria-hidden="true">
  
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://github.com/bitarellolab/" title="github" target="_blank" rel="noopener">
      <i class="fab fa-github fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://twitter.com/dudutchy" title="twitter" target="_blank" rel="noopener">
      <i class="fab fa-twitter fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://scholar.google.com/citations?user=4N3s92MAAAAJ&amp;hl=en" title="google-scholar" target="_blank" rel="noopener">
      <i class="ai ai-google-scholar fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://twitter.com/dudutchy" title="twitter" target="_blank" rel="noopener">
      <i class="fab fa-twitter fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://impactstory.org/u/0000-0001-7676-9367" title="impactstory" target="_blank" rel="noopener">
      <i class="ai ai-impactstory fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://www.youtube.com/channel/UCuZ2G_mWD7_rX9rFoHv0E2w" title="youtube" target="_blank" rel="noopener">
      <i class="fab fa-youtube fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://www.linkedin.com/in/barbarabitarello/" title="linkedin" target="_blank" rel="noopener">
      <i class="fab fa-linkedin fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://orcid.org/0000-0001-7676-9367" title="orcid" target="_blank" rel="noopener">
      <i class="ai ai-orcid fa-lg fa-fw"></i>
    </a>
  
</div>

    </div>
    
    <div class="site-links f6 db dtc-l v-mid w-100 w-67-l tc tr-l pv2 pv0-l mv0">
      
      <a class="dib pv1 ph2 link" href="/license/" title="License">license</a>
      
    </div>
  </nav>
  
    <script>

    var i, text, code, codes = document.getElementsByTagName('code');
    for (let i = 0; i < codes.length;) {
      code = codes[i];
      if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
        text = code.textContent;
        if (/^\$[^$]/.test(text) && /[^$]\$$/.test(text)) {
          text = text.replace(/^\$/, '\\(').replace(/\$$/, '\\)');
          code.textContent = text;
        }
        if (/^\\\((.|\s)+\\\)$/.test(text) ||
            /^\\\[(.|\s)+\\\]$/.test(text) ||
            /^\$(.|\s)+\$$/.test(text) ||
            /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
          code.outerHTML = code.innerHTML;  
          continue;
        }
      }
      i++;
    }
</script>

  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.css" integrity="sha384-RZU/ijkSsFbcmivfdRBQDtwuwVqK7GMOw6IMvKyeWL2K5UAlyp6WonmB8m7Jd0Hn" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.js" integrity="sha384-pK1WpvzWVBQiP0/GjnvRxV4mOb0oxFuyRxJlk6vVw146n3egcN5C925NCP7a7BY8" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/contrib/auto-render.min.js" integrity="sha384-vZTG03m+2yp6N6BNi5iM4rW4oIwk5DfcNdFfxkk9ZWpDriOkXX8voJBFrAO7MpVl" crossorigin="anonymous" onload="renderMathInElement(document.body);"></script>



    
  
  
</footer>

      </div>
    </body>
</html>
