<!DOCTYPE html>
<html lang="en" dir="ltr"><head>
  
                           
     


<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="generator" content="Hugo 0.110.0">
<title>Polygenic risk scores perform poorly across populations | B치rbara Bitarello</title>


<meta property="twitter:site" content="@dudutchy">
<meta property="twitter:creator" content="@dudutchy">







  
    
  
<meta name="description" content="Bitarello lab @ BMC">


<meta property="og:site_name" content="B치rbara Bitarello">
<meta property="og:title" content="Polygenic risk scores perform poorly across populations | B치rbara Bitarello">
<meta property="og:description" content="Bitarello lab @ BMC" />
<meta property="og:type" content="page" />
<meta property="og:url" content="/talk/ashg2018/" />
<meta property="og:locale" content="en">




    
        <meta property="og:image" content="/talk/ashg2018/featured.png" >
        <meta property="twitter:card" content="summary_large_image">
        <meta name="twitter:image" content="/talk/ashg2018/featured.png" >
    
    
  <meta itemprop="name" content="Polygenic risk scores perform poorly across populations">
<meta itemprop="description" content="Abstract | The vast majority of genome-wide association studies (GWAS) are performed in cohorts of European ancestry. Systematic differences in polygenic risk scores (PRS) between European and non-European ancestry populations are believed to be largely spurious. However, it is not clear whether they are completely inaccurate nor how much individual-level predictive power is lost by applying PRS based on European-ancestry GWAS to non-European ancestry populations. Finally, a quantitative understanding of the biological or statistical basis for the poor performance of PRS in non-European-ancestry populations is lacking."><meta itemprop="datePublished" content="2018-10-20T00:00:00+00:00" />
<meta itemprop="dateModified" content="2018-10-20T00:00:00+00:00" />
<meta itemprop="wordCount" content="355"><meta itemprop="image" content="/talk/ashg2018/featured.png">
<meta itemprop="keywords" content="research talk,contributed research talk,PRS,polygenic risk scores,height,precision genomics," />
  
<script type="application/javascript">
var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
var doNotTrack = (dnt == "1" || dnt == "yes");
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'UA-173463950-1', 'auto');
	ga('set', 'anonymizeIp', true);
	ga('send', 'pageview');
}
</script>
  
<script type="application/javascript">
var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
var doNotTrack = (dnt == "1" || dnt == "yes");
if (!doNotTrack) {
	window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
	ga('create', 'UA-173463950-1', 'auto');
	ga('set', 'anonymizeIp', true);
	ga('send', 'pageview');
}
</script>
<script async src='https://www.google-analytics.com/analytics.js'></script>

  <!--[if IE]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  <link rel="shortcut icon" href="/img/dna-solid.svg" type="image/x-icon">
  <link rel="icon" href="/img/dna-solid.svg" type="image/x-icon">
  
  
  <link rel="stylesheet" href="/style.main.min.92dfcd41e6de8ee9bdf4891b6db7aba055136f3e07bb0945cad731e21d441835.css" integrity="sha256-kt/NQebejum99IkbbberoFUTbz4HuwlFytcx4h1EGDU=" media="screen">
  
  
  <script src="/panelset.min.ed1ac24b6e16f4e2481e3d1d098ae66f5bc77438aef619e6e266d8ac5b00dc72.js" type="text/javascript"></script>
  
  
  <script src="/main.min.6407e1256d3f3d21ac8def46942ac52e585729b4c0a5e1f8acc2633c9154b7e1.js" type="text/javascript"></script>
</head>
<body>
      <div class="grid-container">
<header class="site-header pt4 pb2 mb4 bb b--transparent ph5 headroom z-max" role="banner">
  <nav class="site-nav db dt-l w-100" role="navigation">
    <a class="site-brand db dtc-l v-mid link no-underline w-100 w-33-l tc tl-l" href="/" title="Home">
      <img src="/img/dna-solid.svg" class="dib db-l h2 w-auto" alt="B치rbara Bitarello">
    </a>
    <div class="site-links db dtc-l v-mid w-100 w-47-l tc tr-l mt3 mt0-l ttu tracked">
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/about/" title="About us">about</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/people/" title="People">people</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/project/" title="Projects">projects</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/teaching/" title="Teaching">teaching</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 active" href="/talk/" title="Talks, Interviews, etc">talks/press</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/blog/" title="Blog">blog</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/publication/" title="Publications">publications</a>
      
      
    </div>
  </nav>
</header>

<main class="page-main pa4" role="main">
  <section class="page-content mw7 center">
    <article class="post-content pa0 ph4-l">
      <header class="post-header">
        <h1 class="f1 lh-solid measure-narrow mb3 fw4">Polygenic risk scores perform poorly across populations</h1>
        
        
        
      
      <section class="event-details cf pt5-ns pt4">
  
  
  <div class="first-column">
    <h5 class="row-label">Date</h5>
  </div>
  <div class="second-column">
    <p class="row-details">





  October 20, 2018
</p>
  </div>
  <div class="first-column">
    <h5 class="row-label">Time</h5>
  </div>
  <div class="second-column">
    <p class="row-details">




  12:00 AM

</p>
  </div>
  
  
  <div class="first-column">
    <h5 class="row-label">Location</h5>
  </div>
  <div class="second-column">
    <p class="row-details">San Diego, California</p>
  </div>
  
  
  <div class="first-column">
    <h5 class="row-label">Event</h5>
  </div>
  <div class="second-column">
      <p class="row-details">
        <a href="http://www.ashg.org/2018meeting/listing/NumberedSessions.shtml" target="_blank">
          Annual Meeting for the American Society of Human Genetics</a>
      </p>
  </div>
  
</section>
      </header>
      <section class="post-body pt5 pb4">
        



<h3 id="abstract-">Abstract |
  <a href="#abstract-"><svg class="anchor-symbol" aria-hidden="true" height="26" width="26" viewBox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
      <path d="M0 0h24v24H0z" fill="currentColor"></path>
      <path d="M3.9 12c0-1.71 1.39-3.1 3.1-3.1h4V7H7c-2.76.0-5 2.24-5 5s2.24 5 5 5h4v-1.9H7c-1.71.0-3.1-1.39-3.1-3.1zM8 13h8v-2H8v2zm9-6h-4v1.9h4c1.71.0 3.1 1.39 3.1 3.1s-1.39 3.1-3.1 3.1h-4V17h4c2.76.0 5-2.24 5-5s-2.24-5-5-5z"></path>
    </svg></a>
</h3>
<p>The vast majority of genome-wide association studies (GWAS) are performed in cohorts of European ancestry. Systematic differences in polygenic risk scores (PRS) between European and non-European ancestry populations are believed to be largely spurious. However, it is not clear whether they are completely inaccurate nor how much individual-level predictive power is lost by applying PRS based on European-ancestry GWAS to non-European ancestry populations. Finally, a quantitative understanding of the biological or statistical basis for the poor performance of PRS in non-European-ancestry populations is lacking. To test this, we explored how well PRS predict a well-studied and highly polygenic trait: height. We calculated PRS using 41 sets of independent SNPs based on distance (physical or genetic) or LD clumping and pruning methods. We first compared PRS based on effect sizes from two independent GWAS: GIANT and UK Biobank (UKB), both of which were performed in individuals of European ancestry. We replicate previous observations of population-level differences in PRS, but these results are significantly different depending on datasets and clumping strategies. Depending on clumping strategy, the average difference between 1000 Genomes European and African PRS varies from 0.57-6.75 standard deviations (SD) using GIANT and 0.48-2.16 SD using UKB summary statistics. This dependence on clumping strategy supports the idea that most of these differences are spurious. We then investigated individual-level prediction in ~7,300 African American (AA) and ~7,400 European American (EA) individuals. Using UKB effect sizes, we found that PRS explain ~1.7% of height variation in AA individuals, compared to about 5.5% of variation in EA individuals. In both AA and EA, PRS explains more of the variance in height in individuals with a higher proportion of European ancestry. Interestingly, although we find a significant positive correlation (r=0.262, p=7.3e-115) between PRS and European ancestry among AA individuals, the correlation between height and European ancestry is extremely low (r=-0.02, p=0.056) and genome-wide ancestry explains only 0.05% of the variance in height, confirming that cross-population differences in PRS do not correlate to phenotypic differences. Finally, we evaluated whether local ancestry improves prediction for non-European populations, investigated dependence on other genomic features and extended our model to other phenotypes.</p>

        
        <details closed class="f6 fw7 input-reset">
  <dl class="f6 lh-copy">
    <dt class="fw7">Posted on:</dt>
    <dd class="fw5 ml0">October 20, 2018</dd>
  </dl>
  <dl class="f6 lh-copy">
    <dt class="fw7">Length:</dt>
    <dd class="fw5 ml0">2 minute read, 355 words</dd>
  </dl>
  
  
  
  <dl class="f6 lh-copy">
    <dt class="fw7">Tags:</dt>
    <dd class="fw5 ml0"> <a href="/tags/research-talk">research talk</a>  <a href="/tags/contributed-research-talk">contributed research talk</a>  <a href="/tags/prs">PRS</a>  <a href="/tags/polygenic-risk-scores">polygenic risk scores</a>  <a href="/tags/height">height</a>  <a href="/tags/precision-genomics">precision genomics</a> </dd>
  </dl>
  
  <dl class="f6 lh-copy">
    <dt class="fw7">See Also:</dt>
    
    <dd class="fw5 ml0"><a href="/talk/2022-06-29-ancestry-matters-lack-of-representation-of-human-genetic-diversity-in-genomic-databases/">Ancestry Matters: Lack of Representation of Human Genetic Diversity in Genomic Databases</a></dd>
    
    <dd class="fw5 ml0"><a href="/talk/2022-04-08-ashg-podcast/">Interview: American Society of Human Genetics (ASHG) Trainee Paper Spotlight</a></dd>
    
    <dd class="fw5 ml0"><a href="/talk/ashg-podcast/">American Society of Human Genetics (ASHG) Trainee Paper Spotlight</a></dd>
    
  </dl>
</details>

      </section>
      <footer class="post-footer">
        <div class="post-pagination dt w-100 mt4 mb2">
  
  
    <a class="prev dtc pr2 tl v-top fw6"
    href="/talk/ashg2019/">&larr; Investigating the lack of transferability of polygenic risk scores in cohorts with admixed ancestry</a>
  
  
  
</div>

      </footer>
    </article>
    
  </section>
</main>
<footer class="site-footer pv4 bt b--transparent ph5" role="contentinfo">
  <nav class="db dt-l w-100">
    <p class="site-copyright f7 db dtc-l v-mid w-100 w-33-l tc tl-l pv2 pv0-l mv0 lh-copy">
      &copy; 2025 B치rbara Bitarello, Anywhere
      <span class="middot-divider"></span>
      Made with <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/hugo-apero/" rel="dct:source">Hugo Ap칠ro</a></span>.
      <br />
      
Based on <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/formspree/blogophonic-hugo" rel="dct:source">Blogophonic</a></span> by <a xmlns:cc="http://creativecommons.org/ns#" href="https://formspree.io" property="cc:attributionName" rel="cc:attributionURL">Formspree</a>.
    </p>
    
    <div class="site-social-links db dtc-l v-mid w-100 w-33-l tc pv2 pv0-l mv0">
      <div class="social-icon-links" aria-hidden="true">
  
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://github.com/bitarellolab/" title="github" target="_blank" rel="noopener">
      <i class="fab fa-github fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://scholar.google.com/citations?user=4N3s92MAAAAJ&amp;hl=en" title="google-scholar" target="_blank" rel="noopener">
      <i class="ai ai-google-scholar fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://impactstory.org/u/0000-0001-7676-9367" title="impactstory" target="_blank" rel="noopener">
      <i class="ai ai-impactstory fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://www.youtube.com/channel/UCuZ2G_mWD7_rX9rFoHv0E2w" title="youtube" target="_blank" rel="noopener">
      <i class="fab fa-youtube fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://www.linkedin.com/in/barbarabitarello/" title="linkedin" target="_blank" rel="noopener">
      <i class="fab fa-linkedin fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://orcid.org/0000-0001-7676-9367" title="orcid" target="_blank" rel="noopener">
      <i class="ai ai-orcid fa-lg fa-fw"></i>
    </a>
  
</div>

    </div>
    
    <div class="site-links f6 db dtc-l v-mid w-100 w-67-l tc tr-l pv2 pv0-l mv0">
      
      <a class="dib pv1 ph2 link" href="/license/" title="License">license</a>
      
    </div>
  </nav>
  
    <script>

    var i, text, code, codes = document.getElementsByTagName('code');
    for (let i = 0; i < codes.length;) {
      code = codes[i];
      if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
        text = code.textContent;
        if (/^\$[^$]/.test(text) && /[^$]\$$/.test(text)) {
          text = text.replace(/^\$/, '\\(').replace(/\$$/, '\\)');
          code.textContent = text;
        }
        if (/^\\\((.|\s)+\\\)$/.test(text) ||
            /^\\\[(.|\s)+\\\]$/.test(text) ||
            /^\$(.|\s)+\$$/.test(text) ||
            /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
          code.outerHTML = code.innerHTML;  
          continue;
        }
      }
      i++;
    }
</script>

  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.css" integrity="sha384-RZU/ijkSsFbcmivfdRBQDtwuwVqK7GMOw6IMvKyeWL2K5UAlyp6WonmB8m7Jd0Hn" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.js" integrity="sha384-pK1WpvzWVBQiP0/GjnvRxV4mOb0oxFuyRxJlk6vVw146n3egcN5C925NCP7a7BY8" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/contrib/auto-render.min.js" integrity="sha384-vZTG03m+2yp6N6BNi5iM4rW4oIwk5DfcNdFfxkk9ZWpDriOkXX8voJBFrAO7MpVl" crossorigin="anonymous" onload="renderMathInElement(document.body);"></script>



    
  
  
</footer>

      </div>
    </body>
</html>
