<!DOCTYPE html>
<html lang="en" dir="ltr"><head>
  
                           
     


<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="generator" content="Hugo 0.110.0">
<title>Publications | Bárbara Bitarello</title>


<meta property="twitter:site" content="@dudutchy">
<meta property="twitter:creator" content="@dudutchy">







  
    
  
<meta name="description" content="This is home for my research publications, each linked with accompanying preprints, posters, and/or news pieces. &lt;br/&gt;&lt;br/&gt;&lt;i class=&#34;fas fa-file-pdf&#34;&gt;&lt;/i&gt; [Full CV here](/files/bitarello_CV.pdf)
">


<meta property="og:site_name" content="Bárbara Bitarello">
<meta property="og:title" content="Publications | Bárbara Bitarello">
<meta property="og:description" content="This is home for my research publications, each linked with accompanying preprints, posters, and/or news pieces. &lt;br/&gt;&lt;br/&gt;&lt;i class=&#34;fas fa-file-pdf&#34;&gt;&lt;/i&gt; [Full CV here](/files/bitarello_CV.pdf)
" />
<meta property="og:type" content="website" />
<meta property="og:url" content="/publication/" />
<meta property="og:locale" content="en">




    
        <meta property="og:image" content="/publication/sidebar-listing.jpg" >
        <meta property="twitter:card" content="summary_large_image">
        <meta name="twitter:image" content="/publication/sidebar-listing.jpg" >
    
    
  <meta itemprop="name" content="Publications">
<meta itemprop="description" content="This is home for my research publications, each linked with accompanying preprints, posters, and/or news pieces. &lt;br/&gt;&lt;br/&gt;&lt;i class=&#34;fas fa-file-pdf&#34;&gt;&lt;/i&gt; [Full CV here](/files/bitarello_CV.pdf)
">
  
<script type="application/javascript">
var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
var doNotTrack = (dnt == "1" || dnt == "yes");
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'UA-173463950-1', 'auto');
	ga('set', 'anonymizeIp', true);
	ga('send', 'pageview');
}
</script>
  
<script type="application/javascript">
var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
var doNotTrack = (dnt == "1" || dnt == "yes");
if (!doNotTrack) {
	window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
	ga('create', 'UA-173463950-1', 'auto');
	ga('set', 'anonymizeIp', true);
	ga('send', 'pageview');
}
</script>
<script async src='https://www.google-analytics.com/analytics.js'></script>

  <!--[if IE]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  <link rel="shortcut icon" href="/img/dna-solid.svg" type="image/x-icon">
  <link rel="icon" href="/img/dna-solid.svg" type="image/x-icon">
  
    <link rel="alternate" type="application/rss+xml" href="/publication/index.xml" title="Bárbara Bitarello" />
  
  
  <link rel="stylesheet" href="/style.main.min.92dfcd41e6de8ee9bdf4891b6db7aba055136f3e07bb0945cad731e21d441835.css" integrity="sha256-kt/NQebejum99IkbbberoFUTbz4HuwlFytcx4h1EGDU=" media="screen">
  
  
  <script src="/panelset.min.ed1ac24b6e16f4e2481e3d1d098ae66f5bc77438aef619e6e266d8ac5b00dc72.js" type="text/javascript"></script>
  
  
  <script src="/main.min.4801a886424745271c0e35e9d0f76b5981077f5f545884b1233f686fc35758f5.js" type="text/javascript"></script>
</head>
<body>
      <div class="grid-container list-sidebar">
<header class="site-header pt4 pb2 mb4 bb b--transparent ph5 headroom z-max" role="banner">
  <nav class="site-nav db dt-l w-100" role="navigation">
    <a class="site-brand db dtc-l v-mid link no-underline w-100 w-33-l tc tl-l" href="/" title="Home">
      <img src="/img/dna-solid.svg" class="dib db-l h2 w-auto" alt="Bárbara Bitarello">
    </a>
    <div class="site-links db dtc-l v-mid w-100 w-47-l tc tr-l mt3 mt0-l ttu tracked">
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/about/" title="About us">about</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/people/" title="People">people</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/project/" title="Projects">projects</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/teaching/" title="Teaching">teaching</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/talk/" title="Talks, Interviews, etc">talks/press</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/blog/" title="Blog">blog</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 active" href="/publication/" title="Publications">publications</a>
      
      
    </div>
  </nav>
</header>

<main class="page-main pa4" role="main">
	<section class="blog-intro mw7 center mb5">
		<h2 class="f2 f1-ns lh-solid">Publications</h2>
	</section>
	<section class="blog-content mw7 center">
		   
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/2015-mapping-bias-overestimates-reference/" class="db">Mapping bias overestimates reference allele frequencies at the HLA genes in the 1000 genomes project phase I data</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				G3: Genes, Genomes, Genetics
				<strong> (2015)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> D.Y.C. Brandt, V.R.C. Aguiar, B.D. Bitarello, K. Nunes, J. Goudet, D. Meyer
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/1000-genomes">1000 Genomes</a>  <a href="/tags/hla">HLA</a>  <a href="/tags/mapping-bias">Mapping bias</a>  <a href="/tags/ngs">NGS</a> </div>
				</div>
								
				
				 
				<p class="f6 db mv2">© 2015 Brandt et al.Next-generation sequencing (NGS) technologies have become the standard for data generation in studies of population genomics, as the 1000 Genomes Project (1000G). However, these techniques are known to be problematic when applied to highly polymorphic genomic regions, such as the human leukocyte antigen (HLA) genes. Because accurate genotype calls and allele frequency estimations are crucial to population genomics analyses, it is important to assess the reliability of NGS data. Here, we evaluate the reliability of genotype calls and allele frequency estimates of the single-nucleotide polymorphisms (SNPs) reported by 1000G (phase I) at five HLA genes (HLA-A, -B, -C, -DRB1, and -DQB1). We take advantage of the availability of HLA Sanger sequencing of 930 of the 1092 1000G samples and use this as a gold standard to benchmark the 1000G data. We document that 18.6% of SNP genotype calls in HLA genes are incorrect and that allele frequencies are estimated with an error greater than ±0.1 at approximately 25% of the SNPs in HLA genes. We found a bias toward overestimation of reference allele frequency for the 1000G data, indicating mapping bias is an important cause of error in frequency estimation in this dataset. We provide a list of sites that have poor allele frequency estimates and discuss the outcomes of including those sites in different kinds of analyses. Because the HLA region is the most polymorphic in the human genome, our results provide insights into the challenges of using of NGS data at other genomic regions of high diversity.</p>
				 
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://doi.org/10.1534/g3.114.015784" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Publication</a>

</p>
				
			</footer>
		</article>

		 
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/2015-kiwi-genome/" class="db">Kiwi genome provides insights into evolution of a nocturnal lifestyle</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				Genome Biology
				<strong> (2015)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> D. Le Duc, G. Renaud, A. Krishnan, M.S. Almén, L. Huynen, S.J. Prohaska, M. Ongyerth, B.D. Bitarello, H.B. Schiöth, M. Hofreiter, P.F. Stadler, K. Prüfer, D. Lambert, J. Kelso, T. Schöneberg
			</p>

			<footer>
								
				
				 
				<p class="f6 db mv2">Kiwi, comprising five species from the genus Apteryx, are endangered, ground-dwelling bird species endemic to New Zealand. They are the smallest and only nocturnal representatives of the ratites. The timing of kiwi adaptation to a nocturnal niche and the genomic innovations, which shaped sensory systems and morphology to allow this adaptation, are not yet fully understood. Results: We sequenced and assembled the brown kiwi genome to 150-fold coverage and annotated the genome using kiwi transcript data and non-redundant protein information from multiple bird species. We identified evolutionary sequence changes that underlie adaptation to nocturnality and estimated the onset time of these adaptations. Several opsin genes involved in color vision are inactivated in the kiwi. We date this inactivation to the Oligocene epoch, likely after the arrival of the ancestor of modern kiwi in New Zealand. Genome comparisons between kiwi and representatives of ratites, Galloanserae, and Neoaves, including nocturnal and song birds, show diversification of kiwi&#39;s odorant receptors repertoire, which may reflect an increased reliance on olfaction rather than sight during foraging. Further, there is an enrichment of genes influencing mitochondrial function and energy expenditure among genes that are rapidly evolving specifically on the kiwi branch, which may also be linked to its nocturnal lifestyle. Conclusions: The genomic changes in kiwi vision and olfaction are consistent with changes that are hypothesized to occur during adaptation to nocturnal lifestyle in mammals. The kiwi genome provides a valuable genomic resource for future genome-wide comparative analyses to other extinct and extant diurnal ratites.</p>
				 
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://doi.org/10.1186/s13059-015-0711-4" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Publication</a>

</p>
				
			</footer>
		</article>

		 
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/2015-hla-supertype-variations-across-populations/" class="db">HLA supertype variation across populations: new insights into the role of natural selection in the evolution of HLA-A and HLA-B polymorphisms</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				Immunogenetics
				<strong> (2015)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> R. dos Santos Francisco, S. Buhler, J.M. Nunes, B.D. Bitarello, G.S. França, D. Meyer, A. Sanchez-Mazas
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/adaptation">adaptation</a>  <a href="/tags/hla">HLA</a>  <a href="/tags/human-populations">Human Populations</a>  <a href="/tags/natural-selection">Natural Selection</a> </div>
				</div>
								
				
				 
				<p class="f6 db mv2">Supertypes are groups of human leukocyte antigen (HLA) alleles which bind overlapping sets of peptides due to sharing specific residues at the anchor positions—the B and F pockets—of the peptide-binding region (PBR). HLA alleles within the same supertype are expected to be functionally similar, while those from different supertypes are expected to be functionally distinct, presenting different sets of peptides. In this study, we applied the supertype classification to the HLA-A and HLA-B data of 55 worldwide populations in order to investigate the effect of natural selection on supertype rather than allelic variation at these loci. We compared the nucleotide diversity of the B and F pockets with that of the other PBR regions through a resampling procedure and compared the patterns of within-population heterozygosity (He) and between-population differentiation (GST) observed when using the supertype definition to those estimated when using randomized groups of alleles. At HLA-A, low levels of variation are observed at B and F pockets and randomized He and GST do not differ from the observed data. By contrast, HLA-B concentrates most of the differences between supertypes, the B pocket showing a particularly high level of variation. Moreover, at HLA-B, the reassignment of alleles into random groups does not reproduce the patterns of population differentiation observed with supertypes. We thus conclude that differently from HLA-A, for which supertype and allelic variation show similar patterns of nucleotide diversity within and between populations, HLA-B has likely evolved through specific adaptations of its B pocket to local pathogens.</p>
				 
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://doi.org/10.1007/s00251-015-0875-9" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Publication</a>

</p>
				
			</footer>
		</article>

		 
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/2011-intercruzmento-de-humanos/" class="db">Intercruzmento de humanos modernos e neandertais: novas perspectivas a luz da genética</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				Revista da Biologia
				<strong> (2011)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> Bárbara Bitarello, Diogo Meyer
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/dna-antigo">DNA antigo</a>  <a href="/tags/sequenciamento">sequenciamento</a> </div>
				</div>
								
				
				  
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://www.revistas.usp.br/revbiologia/article/view/108622/106943" target="_blank" rel="noopener"><i class="fa fa-file-pdf fa-lg fa-fw mr2"></i>Publication</a>

</p>
				
			</footer>
		</article>

		 
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/2009-projeto-genografico/" class="db">Projeto Genográfico e as Implicações da Popularização dos Estudos de Genealogia Gênica</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				Revista da Biologia
				<strong> (2009)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> Bárbara Domingues Bitarello
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/dna">dna</a>  <a href="/tags/gene-genealogy">gene genealogy</a>  <a href="/tags/genographic">genographic</a> </div>
				</div>
								
				
				 
				<p class="f6 db mv2">O Projeto Genográfico busca caracterizar a variabilidade genética da população mundial, baseando-se majoritariamente em dados genéticos de grupos indígenas isolados e, em menor escala, na participação voluntária de indivíduos interessados em entender suas origens. No entanto, as implicações deste conhecimento merecem ressalva, assim como é importante avaliar o grau de ilusão que está sendo gerado entre os leigos que buscam conhecer sua ancestralidade genética.</p>
				 
			</footer>
		</article>

		 
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/2009-development-of-polymorphic/" class="db">Development of polymorphic microsatellite markers for the human botfly, Dermatobia hominis (Diptera: Oestridae)</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				Molecular Ecology Resources
				<strong> (2009)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> B.D. Bitarello, T.T. Torres, M.L. Lyra, A.M.L. De Azeredo-Espin
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/dermatobia-hominis">Dermatobia hominis</a>  <a href="/tags/livestock-pest">Livestock pest</a>  <a href="/tags/microsatellite">Microsatellite</a>  <a href="/tags/myiasis">Myiasis</a>  <a href="/tags/oestridae">Oestridae</a>  <a href="/tags/population-genetics">Population genetics</a> </div>
				</div>
								
				
				 
				<p class="f6 db mv2">In this report, we describe the development of 17 polymorphic microsatellite markers for the human botfly, Dermatobia hominis, an obligatory parasite of mammals of great veterinary importance in Latin America. The number of alleles ranged from 5 to 21 per locus, with a mean of 12.2 alleles per locus. The expected heterozygosity ranged from 0.2571 to 0.9206 and from 0.2984 to 0.9291 in two populations from Brazil. These markers should provide a high resolution tool for assessment of the fine-scale genetic structure of natural populations of the human botfly.</p>
				 
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://doi.org/10.1111/j.1755-0998.2008.02454.x" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Publication</a>

</p>
				
			</footer>
		</article>

		 



<div class="list-pagination w-100">
  <ul class="paginate list pl0 tc dt w-100 ma0">
    <li class="dtc w-25 tl v-mid">
    
      <a href="/publication/" class="previous pa2 db fw6" aria-label="Previous">&larr; Newer</a>
    
    </li>
    <li class="dtc w-50 v-mid">
      <span class="page_number fw6 f7 ttu tracked">2 of 2</span>
    </li>
    <li class="dtc w-25 tr v-mid">
    
      <span class="next pa2 db fw6">Older &rarr;</span>
    
    </li>
  </ul>
</div>


	</section>
</main>
<aside class="page-sidebar" role="complementary">
	                       
 


                       
 











  <img src="/publication/sidebar-listing.jpg" class="db ma0">


<div class="blog-info ph4 pt4 pb4 pb0-l">
  

  
  <p class="f6 lh-copy measure"><p>This is home for my research publications, each linked with accompanying preprints, posters, and/or news pieces.</p>
<p><i class="fas fa-file-pdf"></i> 
<a href="/files/bitarello_CV.pdf">Full CV here</a></p>
</p>
  


  
</div>


</aside>
<footer class="site-footer pv4 bt b--transparent ph5" role="contentinfo">
  <nav class="db dt-l w-100">
    <p class="site-copyright f7 db dtc-l v-mid w-100 w-33-l tc tl-l pv2 pv0-l mv0 lh-copy">
      &copy; 2023 Bárbara Bitarello, Anywhere
      <span class="middot-divider"></span>
      Made with <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/hugo-apero/" rel="dct:source">Hugo Apéro</a></span>.
      <br />
      
Based on <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/formspree/blogophonic-hugo" rel="dct:source">Blogophonic</a></span> by <a xmlns:cc="http://creativecommons.org/ns#" href="https://formspree.io" property="cc:attributionName" rel="cc:attributionURL">Formspree</a>.
    </p>
    
    <div class="site-social-links db dtc-l v-mid w-100 w-33-l tc pv2 pv0-l mv0">
      <div class="social-icon-links" aria-hidden="true">
  
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://github.com/bitarellolab/" title="github" target="_blank" rel="noopener">
      <i class="fab fa-github fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://twitter.com/dudutchy" title="twitter" target="_blank" rel="noopener">
      <i class="fab fa-twitter fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://scholar.google.com/citations?user=4N3s92MAAAAJ&amp;hl=en" title="google-scholar" target="_blank" rel="noopener">
      <i class="ai ai-google-scholar fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://twitter.com/dudutchy" title="twitter" target="_blank" rel="noopener">
      <i class="fab fa-twitter fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://impactstory.org/u/0000-0001-7676-9367" title="impactstory" target="_blank" rel="noopener">
      <i class="ai ai-impactstory fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://www.youtube.com/channel/UCuZ2G_mWD7_rX9rFoHv0E2w" title="youtube" target="_blank" rel="noopener">
      <i class="fab fa-youtube fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://www.linkedin.com/in/barbarabitarello/" title="linkedin" target="_blank" rel="noopener">
      <i class="fab fa-linkedin fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://orcid.org/0000-0001-7676-9367" title="orcid" target="_blank" rel="noopener">
      <i class="ai ai-orcid fa-lg fa-fw"></i>
    </a>
  
</div>

    </div>
    
    <div class="site-links f6 db dtc-l v-mid w-100 w-67-l tc tr-l pv2 pv0-l mv0">
      
      <a class="dib pv1 ph2 link" href="/license/" title="License">license</a>
      
    </div>
  </nav>
  
    <script>

    var i, text, code, codes = document.getElementsByTagName('code');
    for (let i = 0; i < codes.length;) {
      code = codes[i];
      if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
        text = code.textContent;
        if (/^\$[^$]/.test(text) && /[^$]\$$/.test(text)) {
          text = text.replace(/^\$/, '\\(').replace(/\$$/, '\\)');
          code.textContent = text;
        }
        if (/^\\\((.|\s)+\\\)$/.test(text) ||
            /^\\\[(.|\s)+\\\]$/.test(text) ||
            /^\$(.|\s)+\$$/.test(text) ||
            /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
          code.outerHTML = code.innerHTML;  
          continue;
        }
      }
      i++;
    }
</script>

  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.css" integrity="sha384-RZU/ijkSsFbcmivfdRBQDtwuwVqK7GMOw6IMvKyeWL2K5UAlyp6WonmB8m7Jd0Hn" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.js" integrity="sha384-pK1WpvzWVBQiP0/GjnvRxV4mOb0oxFuyRxJlk6vVw146n3egcN5C925NCP7a7BY8" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/contrib/auto-render.min.js" integrity="sha384-vZTG03m+2yp6N6BNi5iM4rW4oIwk5DfcNdFfxkk9ZWpDriOkXX8voJBFrAO7MpVl" crossorigin="anonymous" onload="renderMathInElement(document.body);"></script>



    
  
  
</footer>

      </div>
    </body>
</html>
