<!DOCTYPE html>
<html lang="en" dir="ltr"><head>
  
                           
     


<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="generator" content="Hugo 0.110.0">
<title>Publications | Bárbara Bitarello</title>


<meta property="twitter:site" content="@dudutchy">
<meta property="twitter:creator" content="@dudutchy">







  
    
  
<meta name="description" content="This is home for my research publications, each linked with accompanying preprints, posters, and/or news pieces. &lt;br/&gt;&lt;br/&gt;&lt;i class=&#34;fas fa-file-pdf&#34;&gt;&lt;/i&gt; [Full CV here](/files/bitarello_CV.pdf)
">


<meta property="og:site_name" content="Bárbara Bitarello">
<meta property="og:title" content="Publications | Bárbara Bitarello">
<meta property="og:description" content="This is home for my research publications, each linked with accompanying preprints, posters, and/or news pieces. &lt;br/&gt;&lt;br/&gt;&lt;i class=&#34;fas fa-file-pdf&#34;&gt;&lt;/i&gt; [Full CV here](/files/bitarello_CV.pdf)
" />
<meta property="og:type" content="website" />
<meta property="og:url" content="/publication/" />
<meta property="og:locale" content="en">




    
        <meta property="og:image" content="/publication/sidebar-listing.jpg" >
        <meta property="twitter:card" content="summary_large_image">
        <meta name="twitter:image" content="/publication/sidebar-listing.jpg" >
    
    
  <meta itemprop="name" content="Publications">
<meta itemprop="description" content="This is home for my research publications, each linked with accompanying preprints, posters, and/or news pieces. &lt;br/&gt;&lt;br/&gt;&lt;i class=&#34;fas fa-file-pdf&#34;&gt;&lt;/i&gt; [Full CV here](/files/bitarello_CV.pdf)
">
  
<script type="application/javascript">
var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
var doNotTrack = (dnt == "1" || dnt == "yes");
if (!doNotTrack) {
	(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
	(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
	m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
	})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
	ga('create', 'UA-173463950-1', 'auto');
	ga('set', 'anonymizeIp', true);
	ga('send', 'pageview');
}
</script>
  
<script type="application/javascript">
var dnt = (navigator.doNotTrack || window.doNotTrack || navigator.msDoNotTrack);
var doNotTrack = (dnt == "1" || dnt == "yes");
if (!doNotTrack) {
	window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
	ga('create', 'UA-173463950-1', 'auto');
	ga('set', 'anonymizeIp', true);
	ga('send', 'pageview');
}
</script>
<script async src='https://www.google-analytics.com/analytics.js'></script>

  <!--[if IE]><script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  <link rel="shortcut icon" href="/img/dna-solid.svg" type="image/x-icon">
  <link rel="icon" href="/img/dna-solid.svg" type="image/x-icon">
  
    <link rel="alternate" type="application/rss+xml" href="/publication/index.xml" title="Bárbara Bitarello" />
  
  
  <link rel="stylesheet" href="/style.main.min.92dfcd41e6de8ee9bdf4891b6db7aba055136f3e07bb0945cad731e21d441835.css" integrity="sha256-kt/NQebejum99IkbbberoFUTbz4HuwlFytcx4h1EGDU=" media="screen">
  
  
  <script src="/panelset.min.ed1ac24b6e16f4e2481e3d1d098ae66f5bc77438aef619e6e266d8ac5b00dc72.js" type="text/javascript"></script>
  
  
  <script src="/main.min.4801a886424745271c0e35e9d0f76b5981077f5f545884b1233f686fc35758f5.js" type="text/javascript"></script>
</head>
<body>
      <div class="grid-container list-sidebar">
<header class="site-header pt4 pb2 mb4 bb b--transparent ph5 headroom z-max" role="banner">
  <nav class="site-nav db dt-l w-100" role="navigation">
    <a class="site-brand db dtc-l v-mid link no-underline w-100 w-33-l tc tl-l" href="/" title="Home">
      <img src="/img/dna-solid.svg" class="dib db-l h2 w-auto" alt="Bárbara Bitarello">
    </a>
    <div class="site-links db dtc-l v-mid w-100 w-47-l tc tr-l mt3 mt0-l ttu tracked">
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/about/" title="About us">about</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/people/" title="People">people</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/project/" title="Projects">projects</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/teaching/" title="Teaching">teaching</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/talk/" title="Talks, Interviews, etc">talks/press</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 " href="/blog/" title="Blog">blog</a>
      
        
        
        
      <a class="link f6 f5-l dib pv1 ph2 active" href="/publication/" title="Publications">publications</a>
      
      
    </div>
  </nav>
</header>

<main class="page-main pa4" role="main">
	<section class="blog-intro mw7 center mb5">
		<h2 class="f2 f1-ns lh-solid">Publications</h2>
	</section>
	<section class="blog-content mw7 center">
		   
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/bls_2023/" class="db">Inferring Balancing Selection From Genome-Scale Data</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				Genome Biology and Evolution
				<strong> (2023)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> Bárbara D Bitarello, Débora Y C Brandt, Diogo Meyer, Aida M Andrés
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/genotype">genotype</a>  <a href="/tags/human-variation">human variation</a>  <a href="/tags/phenotype">phenotype</a>  <a href="/tags/population-genomics">population genomics</a>  <a href="/tags/natural-selection">natural selection</a>  <a href="/tags/balancing-selection">balancing selection</a>  <a href="/tags/review">review</a> </div>
				</div>
								
				
				 
				<p class="f6 db mv2">The identification of genomic regions and genes that have evolved under natural selection is a fundamental objective in the field of evolutionary genetics. While various approaches have been established for the detection of targets of positive selection, methods for identifying targets of balancing selection, a form of natural selection that preserves genetic and phenotypic diversity within populations, have yet to be fully developed. Despite this, balancing selection is increasingly acknowledged as a significant driver of diversity within populations, and the identification of its signatures in genomes is essential for understanding its role in evolution. In recent years, a plethora of sophisticated methods has been developed for the detection of patterns of linked variation produced by balancing selection, such as high levels of polymorphism, altered allele-frequency distributions, and polymorphism sharing across divergent populations. In this review, we provide a comprehensive overview of classicalnd contemporary methods, offer guidance on the choice of appropriate methods, and discuss the importance of avoiding artifacts and of considering alternative evolutionary processes. The increasing availability of genome-scale datasets holds the potential to assist in the identification of new targets and the quantification of the prevalence of balancing selection, thus enhancing our understanding of its role in natural populations.</p>
				 
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://doi.org/10.1093/gbe/evad032" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Publication</a>

</p>
				
			</footer>
		</article>

		 
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/fads_2023/" class="db">Genome-wide analysis identifies genetic effects on reproductive success and ongoing natural selection at the FADS locus.</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				Nature Human Behavior
				<strong> (2023)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> Iain Mathieson, Felix R Day, Nicola Barban, Felix C Tropf, David Brazel, eQTLgen Consortium, BIOS Consortium, Ahmad Vaez, Natalie van Zuydam, Bárbara D Bitarello, Eugene J Gardner, Evelina T Akimova, Ajuna Azad, Sven Bergmann, Lawrence F Bielak, Dorret I Boomsma, Kristina Bosak, Marco Brumat, Julie E Buring, David Cesarini, Daniel I Chasman, Jorge E Chavarro, Massimiliano Cocca, Maria Pina Concas, FinnGen Study, Lifelines Cohort Study, (...) John R B Perry
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/genotype">genotype</a>  <a href="/tags/human-variation">human variation</a>  <a href="/tags/phenotype">phenotype</a>  <a href="/tags/polygenic-traits">polygenic traits</a>  <a href="/tags/population-genetics">population genetics</a>  <a href="/tags/gwas">GWAS</a>  <a href="/tags/natural-selection">natural selection</a> </div>
				</div>
								
				
				 
				<p class="f6 db mv2">Identifying genetic determinants of reproductive success may highlight mechanisms underlying fertility and identify alleles under present-day selection. Using data in 785,604 individuals of European ancestry, we identified 43 genomic loci associated with either number of children ever born (NEB) or childlessness. These loci span diverse aspects of reproductive biology, including puberty timing, age at first birth, sex hormone regulation, endometriosis and age at menopause. Missense variants in ARHGAP27 were associated with higher NEB but shorter reproductive lifespan, suggesting a trade-off at this locus between reproductive ageing and intensity. Other genes implicated by coding variants include PIK3IP1, ZFP82 and LRP4, and our results suggest a new role for the melanocortin 1 receptor (MC1R) in reproductive biology. As NEB is one component of evolutionary fitness, our identified associations indicate loci under present-day natural selection. Integration with data from historical selection scans highlighted an allele in the FADS1/2 gene locus that has been under selection for thousands of years and remains so today. Collectively, our findings demonstrate that a broad range of biological mechanisms contribute to reproductive success.</p>
				 
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://doi.org/10.1038/s41562-023-01528-6" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Publication</a>

    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://www.biorxiv.org/content/10.1101/2020.05.19.104455v1.abstract" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Preprint</a>

</p>
				
			</footer>
		</article>

		 
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/cox_2021/" class="db">Predicting skeletal stature using ancient DNA</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				American Journal of Physical Anthropology
				<strong> (2021)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> Samantha L Cox, Hannah M. Moots, Jay T. Stock, Andrej Shbat, Bárbara D. Bitarello, Nicole Nicklisch,Kurt W. Alt, Wolfgang Haak, Eva Rosenstock, Christopher B. Ruff &amp; Iain Mathieson
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/genotype">genotype</a>  <a href="/tags/human-variation">human variation</a>  <a href="/tags/phenotype">phenotype</a>  <a href="/tags/polygenic-traits">polygenic traits</a>  <a href="/tags/population-genetics">population genetics</a>  <a href="/tags/skeletal-stature">skeletal stature</a> </div>
				</div>
								
				
				 
				<p class="f6 db mv2">Ancient DNA provides an opportunity to separate the genetic and environmental bases of complex traits by allowing direct estimation of genetic values in ancient individuals. Here, we test whether genetic scores for height in ancient individuals are predictive of their actual height, as inferred from skeletal remains. We estimate the contributions of genetic and environmental variables to observed phenotypic variation as a first step towards quantifying individual sources of morphological variation.</p>
				 
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://onlinelibrary.wiley.com/doi/full/10.1002/ajpa.24426" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Publication</a>

</p>
				
			</footer>
		</article>

		 
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/bitarello-2020/" class="db">Polygenic scores for height in admixed populations</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				G3: Genes, Genomes, Genetics
				<strong> (2020)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> Bárbara D. Bitarello, Iain Mathieson
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/polygenic-risk-scores">polygenic risk scores</a>  <a href="/tags/height">height</a>  <a href="/tags/prs">PRS</a>  <a href="/tags/genetic-ancestry">genetic ancestry</a>  <a href="/tags/prediction">prediction</a>  <a href="/tags/c&#43;t">C&#43;T</a>  <a href="/tags/ldpred">ldpred</a>  <a href="/tags/admixture">admixture</a> </div>
				</div>
								
				
				 
				<p class="f6 db mv2">Polygenic risk scores (PRS) use the results of genome-wide association studies (GWAS) to predict quantitative phenotypes or disease risk at an individual level. This provides a potential route to the use of genetic data in personalized medical care. However, a major barrier to the use of PRS is that the majority of GWAS come from cohorts of European ancestry. The predictive power of PRS constructed from these studies is substantially lower in non-European ancestry cohorts, although the reasons for this are unclear. To address this question, we investigate the performance of PRS for height in cohorts with admixed African and European ancestry, allowing us to evaluate ancestry-related differences in PRS predictive accuracy while controlling for environment and cohort differences. We first show that that the predictive accuracy of height PRS increases linearly with European ancestry and is largely explained by European ancestry segments of the admixed genomes. We show that differences in allele frequencies, recombination rate, and marginal effect sizes across ancestries all contribute to the decrease in predictive power, but none of these effects explain the decrease on its own. Finally, we demonstrate that prediction for admixed individuals can be improved by using a linear combination of PRS that includes ancestry-specific effect sizes, although this approach is at present limited by the small size of non-European ancestry discovery cohorts.</p>
				 
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://www.g3journal.org/content/10/11/4027" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Publication</a>

</p>
				
			</footer>
		</article>

		 
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/2018-signatures-of-longerm-balancing/" class="db">Signatures of long-term balancing selection in human genomes</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				Genome Biology and Evolution
				<strong> (2018)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> Bárbara Domingues Bitarello Cesare de Filippo, João Carlos Teixeira, Philip Kleinert, Joshua M Schmidt, Diogo Meyer, Aida M Andrés
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/balancing-selection">balancing selection</a>  <a href="/tags/natural-selection">natural selection</a>  <a href="/tags/site-frequency-spectrum">site frequency spectrum</a>  <a href="/tags/neutrality-tests">neutrality tests</a>  <a href="/tags/ncd">NCD</a>  <a href="/tags/statistical-methods">statistical methods</a>  <a href="/tags/human-variation">human variation</a>  <a href="/tags/population-genomics">population genomics</a> </div>
				</div>
								
				
				 
				<p class="f6 db mv2">Balancing selection maintains advantageous diversity in populations through various mechanisms. While extensively explored from a theoretical perspective, an empirical understanding of its prevalence and targets lags behind our knowledge of positive selection. Here we describe the Non-Central Deviation (NCD), a simple yet powerful statistic to detect long-term balancing selection (LTBS) which quantifies how close frequencies are to expectations under LTBS, and provides the basis for a neutrality test. NCD can be applied to single loci or genomic data, to populations with or without known demographic history, and can be implemented considering only polymorphisms (NCD1) or also considering fixed differences (NCD2). Both statistics have very high power to detect LTBS in humans under different frequencies of the balanced allele(s), with NCD2 having the highest power. Applied to genome-wide data from African and European human populations NCD2 shows that, albeit not prevalent, LTBS affects a sizable portion of the genome: about 0.6% of analyzed genomic windows and 0.8% of analyzed positions. These windows overlap about 8% of the protein-coding genes, which interestingly have larger number of transcripts than expected by chance. Significant windows contain 1.6% of the SNPs in the genome, which disproportionally overlap sites within exons and that alter protein sequence, but not putatively regulatory sites. Our catalog of candidates includes known targets of LTBS, but a majority of them are novel. As expected, immune-related genes are among those with the strongest signatures, although most candidates are involved in other biological functions, suggesting that LTBS potentially influences diverse human phenotypes.</p>
				 
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="http://www.biorxiv.org/content/early/2017/07/24/119529" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Preprint</a>

</p>
				
			</footer>
		</article>

		 
		<article class="center mb4 mb5-l">
			<header>
				<h1 class="f3 mt0 mb2 lh-title underline fw4">
					<a href="/publication/2017-genomic-perspective-on-hla/" class="db">A genomic perspective on HLA evolution</a>
				</h1>				
			</header>

			<p class="f4 db mv2">
				Immunogenetics
				<strong> (2017)</strong>
			</p>
			<p class="f5 db mv1">
				<strong>Authors:</strong> D. Meyer, V.R. C. Aguiar, B.D. Bitarello, D.Y. C. Brandt, K. Nunes
			</p>

			<footer>
				
				<div class="f6 lh-copy">
				  <div class="fw5 ml0">Tags:  <a href="/tags/balancing-selection">balancing selection</a>  <a href="/tags/molecular-evolution">molecular evolution</a>  <a href="/tags/genomics">genomics</a>  <a href="/tags/hla">HLA</a>  <a href="/tags/mhc">MHC</a> </div>
				</div>
								
				
				 
				<p class="f6 db mv2">Several decades of research have convincingly shown that classical human leukocyte antigen (HLA) loci bear signatures of natural selection. Despite this conclusion, many questions remain regarding the type of selective regime acting on these loci, the time frame at which selection acts, and the functional connections between genetic variability and natural selection. In this review, we argue that genomic datasets, in particular those generated by next-generation sequencing (NGS) at the population scale, are transforming our understanding of HLA evolution. We show that genomewide data can be used to perform robust and powerful tests for selection, capable of identifying both positive and balancing selection at HLA genes. Importantly, these tests have shown that natural selection can be identified at both recent and ancient timescales. We discuss how findings from genomewide association studies impact the evolutionary study of HLA genes, and how genomic data can be used to survey adaptive change involving interaction at multiple loci. We discuss the methodological developments which are necessary to correctly interpret genomic analyses involving the HLA region. These developments include adapting the NGS analysis framework so as to deal with the highly polymorphic HLA data, as well as developing tools and theory to search for signatures of selection, quantify differentiation, and measure admixture within the HLA region. Finally, we show that high throughput analysis of molecular phenotypes for HLA genes—namely transcription levels—is now a feasible approach and can add another dimension to the study of genetic variation.</p>
				 
				<p class="db pt1">
    
    
    
    
    
    
    
      
      
  <a class="btn-links mr2 ba dib" href="https://doi.org/10.1007/s00251-017-1017-3" target="_blank" rel="noopener"><i class="ai ai-doi fa-lg fa-fw mr2"></i>Publication</a>

</p>
				
			</footer>
		</article>

		 



<div class="list-pagination w-100">
  <ul class="paginate list pl0 tc dt w-100 ma0">
    <li class="dtc w-25 tl v-mid">
    
      <span class="previous pa2 db fw6">&larr; Newer</span>
    
    </li>
    <li class="dtc w-50 v-mid">
      <span class="page_number fw6 f7 ttu tracked">1 of 3</span>
    </li>
    <li class="dtc w-25 tr v-mid">
    
      <a href="/publication/page/2/" class="next pa2 db fw6" aria-label="Next">Older &rarr;</a>
    
    </li>
  </ul>
</div>


	</section>
</main>
<aside class="page-sidebar" role="complementary">
	                       
 


                       
 











  <img src="/publication/sidebar-listing.jpg" class="db ma0">


<div class="blog-info ph4 pt4 pb4 pb0-l">
  

  
  <p class="f6 lh-copy measure"><p>This is home for my research publications, each linked with accompanying preprints, posters, and/or news pieces.</p>
<p><i class="fas fa-file-pdf"></i> 
<a href="/files/bitarello_CV.pdf">Full CV here</a></p>
</p>
  


  
</div>


</aside>
<footer class="site-footer pv4 bt b--transparent ph5" role="contentinfo">
  <nav class="db dt-l w-100">
    <p class="site-copyright f7 db dtc-l v-mid w-100 w-33-l tc tl-l pv2 pv0-l mv0 lh-copy">
      &copy; 2023 Bárbara Bitarello, Anywhere
      <span class="middot-divider"></span>
      Made with <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/hugo-apero/" rel="dct:source">Hugo Apéro</a></span>.
      <br />
      
Based on <span xmlns:dct="http://purl.org/dc/terms/" property="dct:title"><a xmlns:dct="http://purl.org/dc/terms/" href="https://github.com/formspree/blogophonic-hugo" rel="dct:source">Blogophonic</a></span> by <a xmlns:cc="http://creativecommons.org/ns#" href="https://formspree.io" property="cc:attributionName" rel="cc:attributionURL">Formspree</a>.
    </p>
    
    <div class="site-social-links db dtc-l v-mid w-100 w-33-l tc pv2 pv0-l mv0">
      <div class="social-icon-links" aria-hidden="true">
  
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://github.com/bitarellolab/" title="github" target="_blank" rel="noopener">
      <i class="fab fa-github fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://twitter.com/dudutchy" title="twitter" target="_blank" rel="noopener">
      <i class="fab fa-twitter fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://scholar.google.com/citations?user=4N3s92MAAAAJ&amp;hl=en" title="google-scholar" target="_blank" rel="noopener">
      <i class="ai ai-google-scholar fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://twitter.com/dudutchy" title="twitter" target="_blank" rel="noopener">
      <i class="fab fa-twitter fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://impactstory.org/u/0000-0001-7676-9367" title="impactstory" target="_blank" rel="noopener">
      <i class="ai ai-impactstory fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://www.youtube.com/channel/UCuZ2G_mWD7_rX9rFoHv0E2w" title="youtube" target="_blank" rel="noopener">
      <i class="fab fa-youtube fa-lg fa-fw"></i>
    </a>
  
    
    
    
      
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://www.linkedin.com/in/barbarabitarello/" title="linkedin" target="_blank" rel="noopener">
      <i class="fab fa-linkedin fa-lg fa-fw"></i>
    </a>
  
    
    
    
    
    
    
    
      
    
    <a class="link dib h1 w1 ml0 mr2 f6 o-90 glow" href="https://orcid.org/0000-0001-7676-9367" title="orcid" target="_blank" rel="noopener">
      <i class="ai ai-orcid fa-lg fa-fw"></i>
    </a>
  
</div>

    </div>
    
    <div class="site-links f6 db dtc-l v-mid w-100 w-67-l tc tr-l pv2 pv0-l mv0">
      
      <a class="dib pv1 ph2 link" href="/license/" title="License">license</a>
      
    </div>
  </nav>
  
    <script>

    var i, text, code, codes = document.getElementsByTagName('code');
    for (let i = 0; i < codes.length;) {
      code = codes[i];
      if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
        text = code.textContent;
        if (/^\$[^$]/.test(text) && /[^$]\$$/.test(text)) {
          text = text.replace(/^\$/, '\\(').replace(/\$$/, '\\)');
          code.textContent = text;
        }
        if (/^\\\((.|\s)+\\\)$/.test(text) ||
            /^\\\[(.|\s)+\\\]$/.test(text) ||
            /^\$(.|\s)+\$$/.test(text) ||
            /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
          code.outerHTML = code.innerHTML;  
          continue;
        }
      }
      i++;
    }
</script>

  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.css" integrity="sha384-RZU/ijkSsFbcmivfdRBQDtwuwVqK7GMOw6IMvKyeWL2K5UAlyp6WonmB8m7Jd0Hn" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/katex.min.js" integrity="sha384-pK1WpvzWVBQiP0/GjnvRxV4mOb0oxFuyRxJlk6vVw146n3egcN5C925NCP7a7BY8" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.13.13/dist/contrib/auto-render.min.js" integrity="sha384-vZTG03m+2yp6N6BNi5iM4rW4oIwk5DfcNdFfxkk9ZWpDriOkXX8voJBFrAO7MpVl" crossorigin="anonymous" onload="renderMathInElement(document.body);"></script>



    
  
  
</footer>

      </div>
    </body>
</html>
